src/NJsonSchema.CodeGeneration/DefaultEnumNameGenerator.cs

```diff
@@ -34,7 +34,11 @@ public string Generate(int index, string name, object value, JsonSchema4 schema)
                 .Replace(".", "_")
                 .Replace(",", "_")
                 .Replace("#", "_")
+                .Replace("&", "_")
                 .Replace("-", "_")
+                .Replace("'", "_")
+                .Replace("(", "_")
+                .Replace(")", "_")
                 .Replace("\\", "_");
         }

```

Comment:
Hey @RicoSuter, I have reverted the colon back to a hyphen for now so we can get the other special characters in and close this one off. I'm a bit short on time right now so am not able to investigate or implement a more comprehensive solution.

Do you want to proceed just with these special characters?

Reply:
Fine for now. But i agree that a white list would be better. But keeping it as is will yield new issues/PRs here and we can see all strange enum names and might find better solutions to just turn everything into underscores...